templates/stripes/stripe-toc.php

```diff
@@ -0,0 +1,31 @@
+<?php $toc = $this->tableOfContents($scope) ?>
```

Comment:
Where's `$scope` coming from here?

Reply:
The frontend editor sets a `scope` property on the stripe object which looks like `"break,textblock"` by default and means that the toc stripe should include headings from break and textblock stripes. 

The stripe can be configured to only show break stripe headers or textblock stripe headers and in those cases the scope would look like `"break"` or `"textblock"`.

We could make it a json array if you think that'd make more sense?

Reply:
Oh I see, I was missing those variables being populated like that. I think I like an array better, but if we keep a string (also fine) I'd use `explode()` to turn it into an array before trying to parse it in `Renderer.php`, and then `in_array()` or something so you can check for exact matches.

Reply:
Updated to use an array of strings instead and it's much cleaner 🤌